recipes-graphics/flutter-engine/flutter-engine_git.bb

```diff
@@ -16,6 +16,7 @@ REQUIRED_DISTRO_FEATURES = "opengl"
 
 DEPENDS += "\
     zip-native \
+    freetype \
```

Comment:
this should go here since it's directly related to absence of the `fontconfig` option:
```
PACKAGECONFIG[fontconfig] = "--enable-fontconfig,,fontconfig,freetype"
```

Does this repro on x64?

Reply:
In fact, freetype dnt need for built engine in any case, it use all in-tree copies of sources, but at the some place in built script have pkg-config check.
I dnt have x86 machines for check now.

Reply:
`PACKAGECONFIG[fontconfig] = "--enable-fontconfig,,fontconfig,freetype"`
this not solution
https://docs.yoctoproject.org/ref-manual/variables.html#term-PACKAGECONFIG

Reply:
This would be a preferred solution; as issue is aarch64 specific:
```
diff --git a/recipes-graphics/flutter-engine/flutter-engine_git.bb b/recipes-graphics/flutter-engine/flutter-engine_git.bb
index 389cb21..dfcefb6 100644
--- a/recipes-graphics/flutter-engine/flutter-engine_git.bb
+++ b/recipes-graphics/flutter-engine/flutter-engine_git.bb
@@ -19,6 +19,10 @@ DEPENDS += "\
     ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland', '', d)} \
     "
 
+DEPENDS:aarch64 += "\
+    freetype \
+    "
+
 PV = "${FLUTTER_SDK_VERSION}"
 
 S = "${WORKDIR}/src"
```